docker/django/Makefile

```diff
@@ -37,6 +37,7 @@ push: image
 	@if [ $(UNAME) = "x86_64" ]; then \
 	    echo "Architecture is OK. Pushing.";\
 	    docker push $(REPO):$(DOCKER_TAG_PROD);\
+		docker push $(REPO):$(RQ_TAG);\
```

Comment:
I think we should push RQ_LATEST here too.

Reply:
You're right!